README.md

```diff
@@ -111,6 +111,16 @@ To start using your own HTTP caching server simply pass it's hostname as `CIRRUS
 cirrus run --environment CIRRUS_HTTP_CACHE_HOST=http-cache-host.internal:8080
 ```
 
+## Security
+
+Cirrus CLI tries to run in different environments, but in some environments we choose to provide more usability at the cost of a few security trade-offs:
+
+* SELinux
+  * service container that copies the project directory into a per-task Docker volume using `rsync` always runs unconfined
```

Comment:
Do we even need to mention the `rsync` internal and this path? The volume creation part is fully controlled by the CLI and IMO a user intents to provide contents of the working directory to the containers. Maybe only mention caveats for the dirty mode where user defined code will be run agains a mounted directory?

Reply:
I think this is good for transparency, since people who will be reading this SELinux section might be concerned about what actually runs unconfined.

And mentioning `rsync` here helps to explain a possible attack surface.

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/131/commits/02221a1880f1aebbda7077a0ebf437b16524f182.